Deploy/stacks/UK-building-retrofit/StackDeployment/README.md

```diff
@@ -117,21 +116,32 @@ This query retrieves the average energy efficiency ratings for all buildings in
 
 Further query templates can be found in the Deploy/stacks/UK-building-retrofit/StackDeployment/SPARQL queries directory, covering a range of metrics such as energy consumption, fuel poverty rates, and public health data.
 
+## 5. Feature Info Agent
+The [Feature Info Agent] serves as an access point for the visualisation, enabling queries about data for individual features (e.g., specific geographical locations). In this case, the Feature Info Agent retrieves energy performance and structural descriptions for buildings by querying the Ontop endpoint using the building's IRI.  
+
+The queries and config files can be found in [fia_quires]. Navigate to [services], open [feature-info-agent.json] and under the `Mounts` section, modify `Source` based on where the queries and config files are located at. 
```

Comment:
The bit "Navigate to..." is obsolete. Please remove it. Please see my other comments regarding the FIA.

Reply:
Thank you for the reminder, this is addressed in commit [4eb60b4acc6e48d628673b8595c1cd90a6c490f9] and commit [ 840cd640264d6d66e2c1f301372c8b6640399878]